doomgrader.sh

```diff
@@ -2,15 +2,40 @@
 
 set -e
 
-# doomgrader config
+# doomgrader config; EDIT HERE!
 DOOMGRADER_ROOT=~/doomgrader
 STEAM_PATH=~/.steam/steam/steamapps/common
 
 # internal paths
 DOWNLOAD_PATH=$DOOMGRADER_ROOT/files
 DEPOTDOWNLOADER_PATH=$DOOMGRADER_ROOT/depotdownloader
 
-# prompt for steam credentials
+# function for copying files with prompt
+DOOMGRADER_COPY()
```

Comment:
I think this function name can be lower case and doesn't need the `DOOMGRADER_`. Just `copy` should be clear enough. Also, the 2 comments seem redundant. I would just leave 1 above the actual function

Reply:
Okay, will do!